Dashboard/app/js/lib/views/surveys/survey-details-views.js

```diff
@@ -488,8 +493,59 @@ FLOW.QuestionGroupItemView = FLOW.View.extend({
     FLOW.selectedControl.set('selectedForMoveQuestionGroup', null);
   },
 
+  ajaxCall: function(qgId){
+      self = this;
+      $.ajax({
+          url: '/rest/question_groups/' + qgId,
+          type: 'GET',
+          success: function(data) {
+            if (data.question_group.status == "READY") {
+                // reload this question group the Ember way, so the UI is updated
+                FLOW.questionGroupControl.getQuestionGroup(self.content.get('keyId'));
+                // load the questions inside this question group
+                FLOW.questionControl.populateQuestionGroupQuestions(self.content.get('keyId'));
+            } else {
+                // fire the remote check again
+                self.pollQuestionGroupCopy();
+            }
+          },
+          error: function() {
+            console.log("Error in checking ready status survey group copy");
```

Comment:
**If** we want to keep this logging statement it would make more sense to call `console.error` instead.